index.html

```diff
@@ -1,16 +1,139 @@
+            <p>
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Similique ad sapiente ratione ut minus excepturi illum.
+              Exercitationem alias quae reiciendis?
+            </p>
+          </div>
+          <div class="column column-imgs">
+            <img
+              src="https://images.unsplash.com/photo-1557925923-33b27f891f88?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=333&q=80"
+              alt="round cake with white frosting topped with red raspberries"
+            />
+            <img
+              src="https://images.unsplash.com/photo-1474625342403-1b8a2c0f7215?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=751&q=80"
+              alt="closeup photo of cupcakes on round white ceramic plate"
+            />
+          </div>
+          <div class="column column-imgs">
+            <img
+              src="https://images.unsplash.com/photo-1426869884541-df7117556757?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80"
+              alt="close-up photography of cupcakes"
+            />
+            <img
+              src="https://images.unsplash.com/photo-1571115177098-24ec42ed204d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
+              alt="layered cake"
+            />
+          </div>
+        </div>
+      </section>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+      <section id="Ordering">
+        <h2>Special Orders</h2>
+        <div class="special-orders-imgs">
+          <div>
+            <img
+              src="https://images.unsplash.com/photo-1557979619-445218f326b9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80"
+              alt="M&M pack pouring candies on M&m cake's mouth on marble surface"
+            />
+            <h4>Cake 1</h4>
+          </div>
+          <div>
+            <img
+              src="https://images.unsplash.com/photo-1546856139-46fb2b57ff21?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80"
+              alt="flat-lat photography of pink-icing colored rosette cake"
+            />
+            <h4>Cake 2</h4>
+          </div>
+          <div>
+            <img
+              src="https://images.unsplash.com/photo-1519340333755-56e9c1d04579?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80"
+              alt="round red and white cake with cookies toppings"
+            />
+            <h4>Cake 3</h4>
+          </div>
+          <div>
+            <img
+              src="https://images.unsplash.com/photo-1573127275465-722f36cc4d44?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80"
+              alt="chocolate digger construction 2 year old birthday cake"
+            />
+            <h4>Cake 4</h4>
+          </div>
+        </div>
+      </section>
+    </main>
 
-</html>
\ No newline at end of file
+    <footer>
```

Comment:
well done for remembering to separate out your page using header, main and footer! 👍